mongo/client.go

```diff
@@ -663,7 +682,11 @@ func (c *Client) ListDatabases(ctx context.Context, filter interface{}, opts ...
 	return newListDatabasesResultFromOperation(op.Result()), nil
 }
 
-// ListDatabaseNames returns a slice containing the names of all of the databases on the server.
+// ListDatabaseNames performs a listDatabases operation and returns a slice containing the names of all of the databases
+// on the server. This is equivalent to calling ListDatabases with the NameOnly option set to true.
```

Comment:
It's not equivalent because this is a slice of names and the other would be a slice of specifications with only the name field populated.

Reply:
Removed the comment.

Reply:
Also added a note about using an empty filter to include all databases in the result for consistency.